hmwk.js

```diff
@@ -0,0 +1,92 @@
+ // Hello World !
+console.log("Halo,Dunia!");
+console.log("Ciao, mondo!");// Italian
+console.log("Hola, mundo!"); // Spanish
+
+// **Declare a variable age and initialize it with an integer, using these exact steps:**
+
+let age;
+console.log(age); //it will be 31
+age = 31;
+console.log(age);
+age = 32;
+
+//**Declare a variable name and assign a string to it.
+
+console.log();// the variable will be house and the string will be hers
+let house = "hers";
+console.log(house);
+house = "his";
+console.log(house); // the answer will be his
+
+// **How do you round the number 7.25, to the nearest integer (i.e., whole number)?**
+ let number = 7.25;
+ console.log(number);
+
+ let roundedNumber = 7;
+
+ let newNumber = Math.max([number],[roundedNumber]);
+ console.log(newNumber);
+
+ //**Arrays**
+ const colors = ["purple","pink","blue"];
+ console.log(colors);// It will log the colors stated in the brackets.
+
+ const favAnimals =["fish","flamingo","bird"];
+ console.log(favAnimals);
+ favAnimals.push("cat");
+ console.log(favAnimals);
+ //**More strings
+ let myString = "this is a test";
+ console.log(myString.length);
+ //**If x equals 7, and the only other statement is x = x % 3, what would be the new value of x?**
+
+  let x = 11;
+  x = x % 2;
+  console.log(x);
+
+  let y = 9;
+  y = y % 5;
+  console.log(y);
+
+  let z= 5;
+  z = z % 2;
+  console.log(z);
+
+
+     //**Write a program to answer the following questions:
+
+//1
+var myList = [["milk",2],["apple",4],["bread",3],["yogurt",7],["tomato",8]];
+console.log(myList);
+
+//2
+let a = 6 / 0;
+let b = 10 / 0;
+console.log(a === b);
+
+
+const grid = [[0, 1], [1, 1], [2, 1], [3, 1]];
+console.log(grid[2][1]);
+
+//let counter = 0;increment
+
+
+let counter = 0;
+counter = counter + 1;
+console.log(counter);
+counter++;
+console.log(counter);
+console.log(++counter);
+//cat
```

Comment:
It's great you made a `cat` object to hold all the related data below! But the syntax for defining object keys and values doesn't use `=` and `;` like we use for variables. (I know we hadn't covered this much in class yet before you did this homework.)

The correct syntax to declare an object would be:
```
const cat = {
   name: 'Prince',
   bestFriend: 'Thomas',
   sex: "male",
   age: "one"
 }
```

For objects, each of the key-value pairs (like the key `name` and the value `"Prince"`) ends in a comma `,` instead of a semicolon `;`. Also a colon `:` separates the property names from values instead of an equals sign `=`.